DesktopModules/Vanjaro/Core/Library/Entities/Interface/IBlock.cs

```diff
@@ -8,6 +8,7 @@ public interface IBlock : IThemeTemplate
     {
         string Category { get; }
         string Name { get; }
+        bool Visible { get; }
```

Comment:
Does it not need to implement a set so you can change the visibility at runtime?

Reply:
we don't have any requirement to set the value at runtime